pkg/activator/handler/handler_test.go

```diff
@@ -233,11 +240,15 @@ func TestActivationHandler(t *testing.T) {
 			})
 
 			reporter := &fakeReporter{}
+			servingClient := fakeservingclientset.NewSimpleClientset()
+			servingInformerFactory := servinginformers.NewSharedInformerFactory(servingClient, 0*time.Millisecond)
+			revisionInformer := servingInformerFactory.Serving().V1alpha1().Revisions()
 			handler := ActivationHandler{
 				Activator: act,
 				Transport: rt,
 				Logger:    TestLogger(t),
 				Reporter:  reporter,
+				Throttler: activator.NewThrottler(1000, 1000, 1000, revisionInformer, TestLogger(t)),
```

Comment:
The throttler int values would be better taken in as a struct argument as this is difficult to understand what is being set.

Reply:
yeap, Joe had a similar suggestion in the previous pr